x-pack/plugins/infra/public/pages/metrics/hosts/hooks/use_hosts_table.tsx

```diff
@@ -221,6 +232,28 @@ export const useHostsTable = () => {
           },
         ],
       },
+      ...(displayAlerts
+        ? [
+            {
+              name: TABLE_COLUMN_LABEL.alertsCount,
+              field: 'alertsCount',
+              sortable: true,
+              'data-test-subj': 'hostsView-tableRow-title',
```

Comment:
duplicate data-test-subj :)
```suggestion
              'data-test-subj': 'hostsView-tableRow-alertsCount',
```